src/pages/index.tsx

```diff
@@ -14,65 +14,241 @@ const Home: NextPageWithLayout = () => {
-          className="object-cover object-center"
+          className="object-cover object-center h-full z-0"
         />
+        <a href="https://www.vectorstock.com/royalty-free-vector/realistic-template-notepad-blank-cover-design-vector-23930574">
+          Vector image by VectorStock / Elnur
+        </a>
+        <div className="relative border-1 border-black border-0 h-3/4 w-5/6 top-1/2 left-1/2 -translate-x-1/2 -translate-y-2/3">
+          <div>
+            <Image
+              src="/images/circle_images_big.png"
+              alt="circle images"
+              fill={true}
+              className="object-contain object-center"
+            />
+          </div>
+        </div>
+        <div className="big green curve absolute bottom-0 w-full">
+          <svg
+            width="100%"
+            height="100%"
+            viewBox="0 0 2078 102"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <g filter="url(#filter0_d_2221_282)">
+              <path
+                d="M11 101.312C810.566 -3.80178 1260.62 -7.72194 2067 101.312H11Z"
+                fill="#098D85"
+              />
+              <path
+                d="M11 101.312C810.566 -3.80178 1260.62 -7.72194 2067 101.312H11Z"
+                stroke="#098D85"
+              />
+            </g>
+            <defs>
+              <filter
+                id="filter0_d_2221_282"
+                x="0.934814"
+                y="0.5"
+                width="2076.13"
+                height="101.312"
+                filterUnits="userSpaceOnUse"
+                color-interpolation-filters="sRGB"
+              >
+                <feFlood flood-opacity="0" result="BackgroundImageFix" />
+                <feColorMatrix
+                  in="SourceAlpha"
+                  type="matrix"
+                  values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+                  result="hardAlpha"
+                />
+                <feOffset dy="-10" />
+                <feGaussianBlur stdDeviation="5" />
+                <feComposite in2="hardAlpha" operator="out" />
+                <feColorMatrix
+                  type="matrix"
+                  values="0 0 0 0 0.301961 0 0 0 0 0.576471 0 0 0 0 0.560784 0 0 0 0.3 0"
+                />
+                <feBlend
+                  mode="normal"
+                  in2="BackgroundImageFix"
+                  result="effect1_dropShadow_2221_282"
+                />
+                <feBlend
+                  mode="normal"
+                  in="SourceGraphic"
+                  in2="effect1_dropShadow_2221_282"
+                  result="shape"
+                />
+              </filter>
+            </defs>
+          </svg>
+        </div>
       </div>
-      <div className="relative z-10 -mt-16 flex w-full justify-center">
+
+      <section className="big green area relative w-full min-h-[40rem] sm:min-h-[50rem] border-blue-500 border-0 bg-primary-600 bottom-[1px]">
```

Comment:
Instead of putting debug class names like `big green area`, I suggest adding id attributes instead since `className` is more reserved for styling because of tailwind. Use `kebab-case` for them